lint-configs/README.md

```diff
@@ -0,0 +1,30 @@
+# StackStorm Lint Configs
+
+This repository contains lint configs for different programming languages and
+tools (flake8, pylint, etc.) used by different StackStorm repositories.
+
+Configs are grouped in sub-directories by programming language.
+
+## Usage
+
+To use those configs, add this repository as a git subtree to the repository
+where you want to utilize those configs. After that is done, update make
+targets (or similar) to correctly pass path to the configs to the tools
+in question.
+
+```bash
+git subtree add --prefix lint-configs https://github.com/StackStorm/lint-configs.git master --squash
```

Comment:
Nope - only when you want to update lint-configs subtree you use pull.

There is no additional interaction required from the end user once the subtree is checked in (unlike with submodule).